src/fileformat/file_format/pe/pe_format.cpp

```diff
@@ -1657,7 +1657,8 @@ void PeFormat::loadResourceNodes(std::vector<const PeLib::ResourceChild*> &nodes
 void PeFormat::loadResources()
 {
 	size_t iconGroupIDcounter = 0;
-	unsigned long long rva = 0, size = 0, imageBase = 0;
+	unsigned long long rva = 0, size = 0;
```

Comment:
While we are at massive refactor towards `std::(u)int<size>_t`, could you also change these two?